.circleci/config.yml

```diff
@@ -46,6 +46,28 @@ jobs:
         command: ./e2e-harness teardown
         when: always
 
+  e2eLocalSubDirTest:
+    working_directory: /home/circleci/.go_workspace/src/github.com/giantswarm/e2e-harness
+    machine: true
+    environment:
+      MINIKUBE_VERSION: v0.25.0
+    steps:
+    - checkout
+
+    - attach_workspace:
+        at: .
+
+    - run: ./e2e-harness localkube
+
+    - run: ./e2e-harness setup --remote=false
+
+    - run: ./e2e-harness test --test-dir=integration/sub-dir
```

Comment:
Nice! I see that you have already added the wiring to the jobs so all good :)

I think it would be better to put the new test in a different directory, subdirs of `integration` were already covered.

Reply:
Ack.